server/migrations/20180727122128-add-refresh-token-column-to-user-model.js

```diff
@@ -3,15 +3,15 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
     queryInterface.addColumn(
-      'User',
+      'Users',
```

Comment:
Why is this changed? It looks like a migration from 2018, so I would expect it would have been run once then.

Reply:
@mariechatfield The old migration files were only relevant as I developed locally, at the time of deployment to staging and prod the models were up to date and we didn't have a  need to run it.

Reply:
That makes sense that staging and prod don't need to run any migrations, but I'm still confused about why we need to change the name of a column that was added in 2018. If the original column used the wrong name, I would expect we'd need a separate migration that removed `User` and added `Users`

Reply:
I am also confused about why this change is relevant. 

Is it that this migration has never been run in staging/prod and it was incorrect?

Reply:
@katz12 @mariechatfield Ah, we never ran the migration scripts that's why the typo went unnoticed. I added them as I built locally but we never ran them.